README.md

```diff
@@ -34,7 +53,8 @@ in your browser.
 
 ### Installing Dependencies
 
-From within **each** folder in the root directory:
+From the root director **and** within `/client`, `/integrationServer`,
```

Comment:
I also think this could be worded a little bit better--maybe something like: "Install NPM modules from the root directory, then also within each folder ('/client', '/integrationServer', '/riskService', '/tripService') by running: ..."

**changed 'tripServer' to 'tripService' here because it looks like Ana will be renaming it